pupil-hardware/pupil-headset.md

```diff
@@ -17,89 +17,65 @@ Pupil ships with a number of additional parts. The below sections provide an ove
-    <div class="Grid-cell">
-      {{< lqip-img src="/images/pupil-hardware/pupil-parts-1.jpg" >}}
-      <p style="padding: 10px;">The world camera comes with two lenses. 60 degree FOV lens (shown on the left) and a wide angle 100 degree FOV lens (shown on the right).</p>
-    </div>
-    <div class="Grid-cell">
-      {{< lqip-img src="/images/pupil-hardware/pupil-parts-3.jpg" >}}
-      <p style="padding: 10px;">All Pupil headsets come with 2 sets of nose pads. You can swap the nose pads to customize the fit.</p>
-    </div>
-  </div>
-</div> -->
+All Pupil headsets come with 2 sets of nose pads. You can swap the nose pads to customize the fit.
 
 <aside class="notice">
 Depending on your configuration, some additional parts may be included with your Pupil headset.
 </aside>
 
 ### Pupil Headset Adjustments
 
-> {{< lqip-img figure-class="img-m" src="/images/pupil-hardware/eye-screw.jpg" >}}
-
-> <p style="text-align: center;">Adjust the ball joint firmness by adjusting the socket screw.</p>
-
-<div class="content-container" style="clear:none;">
-  <div class="Grid Grid--1of3 u-textCenter">
-    <div class="Grid-cell">
-      {{< figure-img figure-class="figure-container" src="/images/pupil-hardware/eyearm.gif" >}}
-      <p style="padding: 10px;">You can <b>slide</b> the eye camera arm along the track.</p>
-    </div>
-    <div class="Grid-cell">
-     {{< figure-img figure-class="figure-container" src="/images/pupil-hardware/eyerotate.gif" >}}
-      <p style="padding: 10px;">You can <b>rotate</b> the eye camera about its ball joint.</p>
-    </div>
-    <div class="Grid-cell">
-      {{< figure-img figure-class="figure-container" src="/images/pupil-hardware/world.gif" >}}
-      <p style="padding: 10px;">You can <b>rotate</b> the world camera to align with your FOV.</p>
-    </div>
-  </div>
-</div>
+A lot of design and engineering thought has gone into getting the ergonomics of the headset just right. It is designed to fit snugly, securely, and comfortably. The headset and cameras can be adjusted to accommodate a wide range of users. 
+
+To ensure a robust eye tracking performance, make sure all the cameras are in focus with a good field of view of your eyes.
+
+#### Slide
+> {{< video-webm src="/videos/headset-adjust/eyecam-slide.webm" >}}
+
+The eye camera arm slides in and out of the headset frame. You can slide the eye camera arm along the track.
+
+#### Rotate
+> {{< video-webm src="/videos/headset-adjust/worldcam-rotate.webm" >}}
+
+You can rotate the world camera up and down to align with your FOV.
+
+#### Rotate
+> {{< video-webm src="/videos/headset-adjust/eyecam-rotate.webm" >}}
+
+The eye camera arm is connected to the eye camera via the ball joint. You can rotate about its ball joint.
+
+#### Screw
+> {{< video-webm src="/videos/headset-adjust/eyecam-screw.webm" >}}
+
+The ball joint controls the firmness of the eye camera. Adjust the socket screw to increase or decrease the firmness.
```

Comment:
You can adjust the set screw to control the movement of the eye camera about the ball joint. We recommend setting the set screw so that you can still move the eye camera by hand but not so loose that the eye camera moves when moving the head. You can also tighten the set screw to fix the eye camera in place.